.buildkite/pipeline.yml

```diff
@@ -15,12 +15,12 @@ steps:
       queue: "juliagpu"
       cuda: "*"
     if: build.message !~ /\[skip tests\]/
-    timeout_in_minutes: 30
+    timeout_in_minutes: 60
     matrix:
       setup:
         julia:
           - "1.10"
-          - "1.11"
+          - "1.12"
```

Comment:
I guess there is no `lts` and `1` on buildkite?

Reply:
Unfortunately, no 😞